### PR TITLE
Add relnotes 2.0

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,26 @@ meta:
...
+### What's new
+
+* **Public API**. You can now use the Chainstack API to manage your resources. See the [API documentation](/api/).
+* **Protocols**.
+  * [Corda 4.5](/blockchains/corda) support.
+  * [Hyperledger Fabric 2.2](/blockchains/fabric) support.
+* **Node management**. You can now [stop and start](/platform/stop-or-start-a-node) your nodes to save your usage costs.
+* **Service nodes**. You can now access in the UI the [service nodes](/glossary/service-node) deployed with your consortium networks.
+* **Clouds**. You can now deploy your nodes and networks in the <a href="https://support.chainstack.com/hc/en-us/articles/360024804711-Data-center-locations" target="_blank">Microsoft Azure UK South</a> region.
+* **Pricing**. No more user limit on all  <a href="https://chainstack.com/pricing/" target="_blank">pricing plans</a>.
```
Comment (easeev) [inline docs/release-notes.md:24]:
```suggestion
* **Pricing**. No more user limit on all <a href="https://chainstack.com/pricing/" target="_blank">pricing plans</a>.
```

File: docs/release-notes.md [hunk 2]
```diff
@@ -8,6 +8,26 @@ meta:
...
+* **Marketplace**. Introducing a <a href="https://chainstack.com/marketplace/" target="_blank">marketplace of solutions</a> to greatly simplify and enhance your blockchain experience.
```
Comment (easeev) [inline docs/release-notes.md:25]: I think Marketplace should not be part of the release notes

File: docs/release-notes.md [hunk 3]
```diff
@@ -8,6 +8,25 @@ meta:
...
+### What's new
+
+* **Public API**. You can now use the Chainstack API to manage your resources. See the [API documentation](/api/).
+* **Protocols**.
+  * [Corda 4.5](/blockchains/corda) support.
+  * [Hyperledger Fabric 2.2](/blockchains/fabric) support.
+* **Node management**. You can now [stop and start](/platform/stop-or-start-a-node) your nodes to save your usage costs.
+* **Service nodes**. You can now access in the UI the [service nodes](/glossary/service-node) deployed with your consortium networks.
+* **Clouds**. You can now deploy your nodes and networks in the <a href="https://support.chainstack.com/hc/en-us/articles/360024804711-Data-center-locations" target="_blank">Microsoft Azure UK South</a> region.
+* **Pricing**. No more user limit on all <a href="https://chainstack.com/pricing/" target="_blank">pricing plans</a>.
```
Comment (easeev) [inline docs/release-notes.md:24]:
```suggestion
* **User management**. No more user limit on all <a href="https://chainstack.com/pricing/" target="_blank">pricing plans</a>.
```

?

Comment (akegaviar) [reply docs/release-notes.md:24]: I strongly feel this change is really about pricing and not user management at all.

**Review by easeev — APPROVED** [review]
LGTM

**Pull request merged**